app/views/sections/show.html.slim

```diff
@@ -24,28 +31,44 @@
           - @latest_videos.each do |video|
             .medium-4.cell
               = link_to(video)
-                = image_tag href="http://lorempixel.com/370/175/animals"
-              - unless video.nil?  
+                = image_tag video.image_url
+              - if video
```

Comment:
Actually, since you're inside a loop, `video` will always be present. So the `if` is not necessary.